source/Octopus.Client.Tests/PublicSurfaceAreaFixture.ThePublicSurfaceAreaShouldNotRegress..NETCore.approved.txt

```diff
@@ -4900,6 +4900,7 @@ Octopus.Client.Model
     Boolean IsDefault { get; set; }
     Boolean IsPrivate { get; set; }

```

Comment:
I suspect there is also the netframework approval, not sure how you generated that last time though.

Reply:
Yeah, [last time](https://github.com/OctopusDeploy/OctopusClients/pull/639) I updated two approved files, this time I was only prompted for one. The test passes whether I include it or not, and it's passed all the tests here. Should I be concerned or just go ahead and merge it?

Reply:
Last time you wouldn't have been prompted for the netframework file - you are on a mac, and so can't run that test. You must have updated it by hand. I would do the same here, update the netframework file by hand to ensure it matches what we would expect to be the output.

Reply:
Ah sure, I guess it was a CI failure that made me update it last time. Which begs the question, why did it pass without it? Anyway, I've updated it it's passed again so I suppose I'll merge this.